aten/src/ATen/native/native_functions.yaml

```diff
@@ -1310,10 +1310,32 @@
     CPU: floor_out
     CUDA: floor_out
 
-- func: floor_divide(Tensor input, Tensor other) -> Tensor
+- func: floor_divide(Tensor self, Tensor other) -> Tensor
+  variants: function, method
+  dispatch:
+    CPU: floor_divide
```

Comment:
why don't you have SparseCPU and SparseCUDA implementations, like in `div`?

Reply:
Let's add some.